README.md

```diff
@@ -36,7 +36,8 @@ using the Docker image.
 
 1. Beginning with Swarm 0.4 golang 1.4.x or later is required for building Swarm. 
```

Comment:
@MHBauer Thanks for the catch. The structure can be tweaked a bit to tell them to actually check the GOPATH .  Also, some other clarity issues:

----> https://gist.github.com/moxiegirl/ef03be443ca6cf05d6b5

### Prerequisites

Beginning with Swarm 0.4, golang 1.4.x or later is required for building Swarm.

1. Refer to the [Go installation page](https://golang.org/doc/install#install)
to download and install the golang 1.4.x or later package.

  > **Note**: On Ubuntu 14.04, the `apt-get` repositories install golang 1.2.1
  version by default. So, do not use `apt-get` but install golang 1.4.x manually
  using the instructions provided on the Go site.

2. After Go is installed, set up a [workspace and
`GOPATH`](https://golang.org/doc/code.html#GOPATH).  

3. Install [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).

4. Install [godep](https://github.com/tools/godep).

  `GOPATH` should be set when you installed `godep`.

5. Verify that `$GOPATH` was set.

        $ echo $GOPATH
        /Users/mary/go

### Get the code and build Swarm

1. Get the source code with the `go get` command.

        $ go get -d github.com/docker/swarm

  This command installs the `docker/swarm` repo in your `$GOPATH`.

2. Change to your `GOPATH`.

        $ cd $GOPATH/src/github.com/docker/swarm/

3. Build and install the binary with the `godep` command.

        $ $GOPATH/bin/godep go install

4. Verify the `swarm` was binary was installed in `$GOPATH/bin`.

        $ ls $GOPATH/bin

To use the `swarm` binary, follow the instructions [in Docker's
documentation](http://docs.docker.com/swarm/), replacing `docker run swarm` with
just `swarm`.